topology/Wayne State University/WSU - GRID_ce2/GRID_ce2.yaml

```diff
@@ -0,0 +1,51 @@
+Disable: false
+Production: true
+SupportCenter: Self Supported
+GroupID: 472
+GroupDescription: The WSU-GRID_ce2 cluster
+Resources:
+  OSG_US_WSU_GRID_ce2:
+    ID: 895
+    Description: The Hosted CE serving WSU-GRID_ce2
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Marco Mascheroni
+          ID: 030408ab932e143859b5f97a2d1c9e30ba2a9f0d
+        Secondary:
+          Name: Jeffrey Michael Dost
+          ID: 3a8eb6436a8b78ca50f7e93bb2a4d1f0141212ba
+      Security Contact:
+        Primary:
+          Name: Marco Mascheroni
+          ID: 030408ab932e143859b5f97a2d1c9e30ba2a9f0d
+        Secondary:
+          Name: Jeffrey Michael Dost
+          ID: 3a8eb6436a8b78ca50f7e93bb2a4d1f0141212ba
+      Site Contact:
+        Primary:
+          ID:  f84214412f7b3dbf848808c4ba30d179c5743766
+          Name: Michael Thompson
+    FQDN: hosted-ce33.grid.uchicago.edu
+    Services:
+      CE:
+        Description: WSU-GRID_ce2 Hosted CE
+        Details:
+          hidden: false
+  OSG_US_WSU_SQUID:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID:  f84214412f7b3dbf848808c4ba30d179c5743766
+          Name: Michael Thompson
+      Security Contact:
+        Primary:
+          ID:  f84214412f7b3dbf848808c4ba30d179c5743766
+          Name: Michael Thompson
+    Description: squid service for frontier and cvmfs
+    FQDN: vsquid1.nas.grid.wayne.edu
+    ID: 1026
+    Services:
+      Squid:
+        Description: Generic squid service
```

Comment:
This looks like a copy-paste error?

Reply:
I was not sure if it needed to be moved to the new resource. Removing it from here for the time being.